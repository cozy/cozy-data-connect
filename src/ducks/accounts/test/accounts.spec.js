/* eslint-env jest */

import { buildAccount } from '../'

describe('Accounts Duck', () => {
  describe('buildAccount', () => {
    const auth = { login: 'testlogin', password: 'testpassword' }

    it('builds an account without name', () => {
      expect(buildAccount(auth)).toMatchSnapshot()
    })

    it('builds an account with a name', () => {
      expect(buildAccount(auth, 'Account Name')).toMatchSnapshot()
    })
  })
})
