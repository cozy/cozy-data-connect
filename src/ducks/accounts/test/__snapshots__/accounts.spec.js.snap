exports[`Accounts Duck buildAccount builds an account with a name 1`] = `
Object {
  "auth": Object {
    "login": "testlogin",
    "password": "testpassword",
  },
  "name": "Account Name",
  "type": "io.cozy.accounts",
}
`;

exports[`Accounts Duck buildAccount builds an account without name 1`] = `
Object {
  "auth": Object {
    "login": "testlogin",
    "password": "testpassword",
  },
  "type": "io.cozy.accounts",
}
`;
